src/Npgsql/TypeHandlers/CompositeHandlers/CompositeConstructorHandler.Generic.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Threading.Tasks;
+using Npgsql.PostgresTypes;
+
+namespace Npgsql.TypeHandlers.CompositeHandlers
+{
+    sealed class CompositeConstructorHandler<TComposite, T1, T2, T3, T4, T5, T6, T7, T8> : CompositeConstructorHandler<TComposite>
+    {
+        delegate TComposite CompositeConstructor(in Arguments args);
+
+        private readonly CompositeConstructor _constructor;
```

Comment:
Remove private (you are also getting infected by EF Core! :) )

Reply:
It's a general style in .NET and .NET Core repos which I'm following (: Will remove this modifier.

Reply:
Yeah, I know, we're a bit special in omitting them - but anything to make the code more terse :)